extensions/azurePublish/src/node/deploy.ts

```diff
@@ -473,11 +472,11 @@ export class BotProjectDeploy {
     }
     try {
       const tenantUrl = `https://management.azure.com/subscriptions/${subId}?api-version=2020-01-01`;
-      const options = {
+      const options: AxiosRequestConfig = {
         headers: { Authorization: `Bearer ${accessToken}` },
       };
-      const response = await rp.get(tenantUrl, options);
-      const jsonRes = JSON.parse(response);
+      const response = await axios.get(tenantUrl, options);
+      const jsonRes = JSON.parse(response?.data);
```

Comment:
this may throw

Reply:
Changed it, it doesn't have to be parsed at all.